hue/templates/configmap.yaml

```diff
@@ -191,15 +138,15 @@ data:
       livy_server_session_kind={{ .Values.config.spark.livyServerSessionKind }}
 
       # Whether Livy requires client to perform Kerberos authentication.
-      security_enabled={{ .Values.config.spark.securityEnabled }}
+      #security_enabled={{ .Values.config.spark.securityEnabled }}
 
       # Host of the Sql Server
-      sql_server_host={{ .Values.config.spark.sqlServerHost }}
+      #sql_server_host={{ .Values.config.spark.sqlServerHost }}
 
       # Port of the Sql Server
-      sql_server_port={{ .Values.config.spark.sqlServerPort }}
+      #sql_server_port={{ .Values.config.spark.sqlServerPort }}
 
       # Choose whether Hue should validate certificates received from the server.
-      ssl_cert_ca_verify={{ .Values.config.spark.sslCaCertVerify }}
+      #ssl_cert_ca_verify={{ .Values.config.spark.sslCaCertVerify }}
```

Comment:
if you want to remove this, you should do it via a conditional

Reply:
Let's default it to false.